web/src/realm_playground.ts

```diff
@@ -24,7 +21,7 @@ export function update_playgrounds(playgrounds_data: RealmPlayground[]): void {
             id: data.id,
             name: data.name,
             url_template: data.url_template,
-            pygments_language: data.pygments_language
+            pygments_language: data.pygments_language,
```

Comment:
This hunk looks like it belongs in the first commit, as do some of the other line-wrapping changes. Please make sure easy commit individually passes lint, it's very difficult to review work where one doesn't do that.

Reply:
Oo no !!!. I am so sorry for it; fixed it now.